Makefile

```diff
@@ -4,10 +4,20 @@
 
 # Author: Thomas Benz <tbenz@iis.ee.ethz.ch>
 
-GIT ?= git
+GIT    ?= git
 BENDER ?= bender
 PYTHON ?= python3
 
+RTL_CFGS ?= \
```

Comment:
@TheMightyDuckOfDoom; make sure that we always call the generator through the makefile. Also in the non-free CI!

Reply:
CI now uses Makefile gen_rtl_* targets